.github/workflows/Build-and-deploy-mac.yml

```
@@ -1,4 +1,4 @@
-name: Build-and-deploy-mac
+name: Build and Deploy — Mac
```

I don't think these will take effect until on `main`

But I'd be in favor of even shorter / more direct

```suggestion
name: Mac Release
```

---

And then 'Dev Tests', 'PyFlask Tests', 'External Tests', and might eventually be useful to separate the testing suites across Python backend, vitetest frontend, and e2e for better reporting/localization of failures

---

Makes sense. Thanks!